tests/testthat/_snaps/posthoc_buffer_rset.md

```diff
@@ -3,7 +3,7 @@
     Code
       suppressMessages(spatial_buffer_vfold_cv(ames_sf, buffer = 500, radius = NULL))
     Condition
-      Error in `wrap_vfold()`:
+      Error in `posthoc_buffer_rset()`:
```

Comment:
I'm sorry I didn't notice before, but we should pass the `call` around here so we get more helpful error, i.e. "Error in `spatial_buffer_vfold_cv()`"